src/Annotations/Serializer/HypothesisClientAnnotationNormalizer.php

```diff
@@ -145,8 +143,8 @@ private function processListBlock(Element\ListBlock $block)
                 foreach ($item->children() as $child) {
                     if ($child instanceof Element\ListBlock) {
                         $items[] = [$render($child)];
-                    } else {
-                        $items[] = $this->htmlRenderer->renderBlock($child);
+                    } elseif ($item = $this->htmlRenderer->renderBlock($child)) {
```

Comment:
Not introduced by this PR, but `processText()` has already rendered the whole list then thrown it away (in `processText()`) in favour of doing each item here?

Reply:
The render of the list converts the whole list to HTML. We need to step through the list to separate out each item. I will see if I can improve this.

Reply:
You just need to stop it rendering the whole thing if it's a list, it's wasted effort.